ebu_tt_live/documents/ebutt3_segmentation.py

```diff
@@ -158,6 +158,7 @@ def _prune_orphan_elements(self):
             self._segment.head.styling = None
         if len(self._segment.head.layout.region) == 0:
             self._segment.head.layout.region = None
```

Comment:
Is this line still needed?

Reply:
hmm that looks strange, like it should say `self._segment.head.layout = None` instead. Good catch.

Reply:
Actually the newly added line just below (line 161) contains the call you mention. The question is whether the current line (160) is still needed, as the new line (161) affects the parent node of line 160's node.

Reply:
Oh yes I see what you mean, it does look like that line can be removed. I'll test removing it and if all goes well I'll push that change. Thank you @spoeschel !

Reply:
Done in 5011a16b3dc790c571234b92391da9712c70a064